IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeotechTypicalSection/DocEntity.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcGeotechTypicalSection" UniqueId="69b8ea2f-0b5a-4686-86a3-23811f54c68a" BaseDefinition="IfcLinearZone" EntityFlags="32">
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="e269468d-3360-41be-8f1d-7feb02283978" DefinedType="IfcGeotechTypicalSectionTypeEnum" AttributeFlags="1">
+			<Documentation>Identifies the predefined type of a geotech typical sectionl element. This type may associate additional specific property sets.</Documentation>
+		</DocAttribute>
+	</Attributes>
+	<WhereRules>
+		<DocWhereRule Name="CorrectPredefinedType" UniqueId="434a01d7-f0fb-4b4a-ba95-067dec9d5639">
```

Comment:
With the above comment in mind we would then also change the rule.

Reply:
I agree.

Reply:
Yes!